### ISSUE TITLE
The car mode is broken

### ISSUE THREAD

**Issue by ahmed-nawar** [stadtnavi/digitransit-ui#709]
https://herrenberg.beta.stadtnavi.eu/reiseplan/Bahnhof%20Herrenberg%2C%2071083%20Herrenberg%3A%3A48.5943214%2C8.8621132/Stuttgart%20Main%20Station%2C%20Stuttgart%3A%3A48.7856099%2C9.1833959?time=1624785707

An error occurs when clicking on the car mode to show details.

**_Search for any route_**
![image](https://user-images.githubusercontent.com/84456072/123539449-1ed0af80-d73a-11eb-845c-3857a7117f80.png)

**_After clicking on the Car mode_**
![image](https://user-images.githubusercontent.com/84456072/123539465-3740ca00-d73a-11eb-8498-b8296af48745.png)

**Comment by leonardehrenfried** [comment]
Yes, this is something that I noticed on Friday, fixed, and immediately deployed straight to prod, skipping beta.

**Comment by MartinH-open** [comment]
Oh - similar can still happen on production as of now: https://herrenberg.stadtnavi.eu/
I just ran into it then selecting the foot tab after using the Ruftaxi tab.

**Comment by ahmed-nawar** [comment]
> Oh - similar can still happen on production as of now: https://herrenberg.stadtnavi.eu/
> I just ran into it then selecting the foot tab after using the Ruftaxi tab.

I cannot reproduce the issue. Can you walk me through it?

**Comment by leonardehrenfried** [comment]
The fix is on beta since last night.

**Comment by MartinH-open** [comment]
> > Oh - similar can still happen on production as of now: https://herrenberg.stadtnavi.eu/
> > I just ran into it then selecting the foot tab after using the Ruftaxi tab.
> 
> I cannot reproduce the issue. Can you walk me through it?

On the beta at this time I can still get into the error situation:
![image](https://user-images.githubusercontent.com/1383111/123632788-a1c83780-d818-11eb-9a82-b54f82bb4c1b.png)

After this the click on the "RufTaxi" tab does not change anything.
And then click on the "walking tab" and you see this
Ein Fehler ist aufgetreten
Nochmal versuchen
There was an error
Try again
![image](https://user-images.githubusercontent.com/1383111/123632866-b99fbb80-d818-11eb-8843-6d9eb2abbf79.png)
This error situation occurs after first the tabs (walking tab, RufTaxi tab) work fine and then you change the Setting "Avoid walking". It appears that the setting change does not lead to a new calculation of the routes.

**Comment by MartinH-open** [comment]
As of now I can get into same error case on https://herrenberg.stadtnavi.eu/ (with a few more clicks)
